packages/app/cypress/component/specs/ResizeObserverException.cy.tsx

```diff
@@ -0,0 +1,37 @@
+import { useElementSize } from '@vueuse/core'
+import { computed, h, ref } from 'vue'
+
+describe('does not fail due to unhandle ResizeObserver error', () => {
```

Comment:
Minimal example to trigger ResizeObserver limit error in Chrome, extracted from one of our specs that has this same problem.